tests/Unit/Domain/Creators/Test_BinaryCompactObjectHelpers.cpp

```diff
@@ -241,6 +267,7 @@ void test(const bool include_expansion, const bool include_rotation,
 
   const std::array<std::array<double, 3>, 2> centers{
       std::array{5.0, 0.01, 0.02}, std::array{-5.0, -0.01, -0.02}};
+  const double domain_envelope_radius = 5.0;
```

Comment:
This radius is within the cubes since their centers are at 5.0. Maybe make this something bigger like 15?